kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+
+  <div>
+
+    <breadcrumbs
+      v-if="$route.name === 'explore-content'">
+    </breadcrumbs>
+    <a
```

Comment:
What I do like, though, is the work you have done to unify the two content pages. Much DRY!